README.md

```diff
@@ -309,8 +309,6 @@ _WARNING: This step requires the working OCI Registry, Cluster and Kyma CLI from
      # Samples Config
      configs:
      ```
-     _WARNING: Even though this file is empty, it is mandatory for the command to succeed as it will be bundled as layer!
```

Comment:
We need more words to explain what's the meaning of this module configuration, and what's the use case.